incubator/catalog-schema-registry/src/main/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/SchemaRegistryCatalogHandler.java

```diff
@@ -49,12 +57,7 @@ public SchemaRegistryCatalogHandler(
         this.crc32c = new CRC32C();
         this.cache = new Int2ObjectCache<>(1, 1024, i -> {});
         this.schemaIdCache = new Int2ObjectCache<>(1, 1024, i -> {});
-    }
-
-    @Override
-    public String type()
-    {
-        return SCHEMA_REGISTRY;
+        this.prefixRO = new UnsafeBuffer(new byte[5]);
```

Comment:
We should probably have this structure generated from an internal `idl` instead for better readability, much like we do for the kafka cache entry descriptions and other protocol codec flyweights.